Code/api/services/mailing.js

```diff
@@ -0,0 +1,130 @@
+
+var User = require('../models/users'),
+    Support = require('../models/support'),
+    nodemailer = require("nodemailer")
+
+
+var headers = {
+    accountPinRecovery: "0xAccountPinRecovery"
+},
+    pinRecoveryMailExpiry = 1000 * 3600 * 1,
+    SMTP_config = {
+        service: "Gmail",
+        auth: {
+            user: 'fiuvipmailer@gmail.com',
+            pass: 'vipadmin123'
+        }
+    },
+    SMTP_Transport = nodemailer	.createTransport("SMTP", SMTP_config);
+  // SMTP_Transport = nodemailer.createTransport('smtps://visualnet2008@gmail.com:20001142h@smtp.gmail.com');
+	//SMTP_Transport = nodemailer.createTransport('smtps://vvega019@fiu.edu:Vega2016-@smtp.gmail.com');
+var CONFIG = {
+    Sender: 'vvega019@fiu.edu' 
+};
+
+exports.sendPinRecoveryCode = function (user_token, SCCallback, ERCallback) {
+
+    if (!SCCallback) SCCallback = function () { }; if (!ERCallback) ERCallback = function () { };
+
+    User.findById(user_token.valueOf(), function (err, user) {
+
+        if (err) {
+            throw err
+        }
+
+        if (user) {
+
+           var object = {}
+            object.user_id = user_token
+            object.header = headers.accountPinRecovery
+            object.verifyCode = _CREATE__VERIFICATION___CODE().toString()
+            object.date = Date.now()
+
+            /*var content = "http://45.32.254.22:3000/#/password_request?auth_id=" + object.user_id + "&code=" + object.verifyCode*/
+            var content = "Dear VIP user, you received this email in order to a password reset request. Please click on the link below: http://localhost:3000/#/password_request?auth_id=" + object.user_id + "&code=" + object.verifyCode

```

Comment:
http://localhost:3000/#/password_request?auth_id=" + object.user_id + "&code=" + object.verifyCode

 should not be localhost... should be vip-dev address